### DISCUSSION TITLE
Official Mastodon Or NOT?

### DISCUSSION THREAD

**Discussion by colinculbreth** [mastodon/mastodon#21086] [Q&A] [answered]
Hi, Twitter has a handle called @mastodonsupport. I assume it isn't a real support, but it did offer me help initially. I tweeted them a few weeks later and I got this response (see attached image)
![IMG_9785](https://user-images.githubusercontent.com/118605096/202847698-866b3eff-832f-4647-857f-b33e790c8fd3.PNG)

Can someone officially working with Mastodon confirm if this is a legit account? If not, I suggest aggressively getting it suspended.

**Comment by verovaleros** [comment]
Adding to this that the account does not look legit: 
https://twitter.com/MastodonSupport/status/1591894178757509121
<img width="473" alt="image" src="https://user-images.githubusercontent.com/2458879/202848046-57098de0-d44c-489d-8c8d-a6e425af4a7f.png">

**Comment by Tslmuun** [comment, marked as answer]
Official Mastodon account is https://twitter.com/joinmastodon
